client/slack_test.go

```diff
@@ -177,7 +177,43 @@ func TestIncomingMessages(t *testing.T) {
 		assert.Equal(t, "Foox", payload.User.LastName)
 		assert.Equal(t, "now", payload.Timestamp)
 		assert.Equal(t, "thread", payload.ThreadTimestamp)
+		assert.Equal(t, "thread", payload.Thread)
+	default:
+		assert.Fail(t, "expected message event")
+	}
+
+	//Test whether Thread is populated correctly
```

Comment:
Maybe you can use test tables here. There is no need to copy and paste the same code.

Reply:
👍  will do